mapper/mapper_test.go

```diff
@@ -481,6 +481,7 @@ func TestUnitMapper(t *testing.T) {
 			Convey("CreateBulletinModel maps correctly", func() {
 				requestProtocol := "https"
 				model := CreateBulletinModel(basePage, bulletin, breadcrumbs, "cy", requestProtocol, serviceMessage, bannerData)
+				preGTMJavaScript := createPreGTMJavaScript(model.Metadata.Title, model)
```

Comment:
Isn't this calling actual code to create the expected response?? There is not value in that, it isn't testing anything?